### ISSUE TITLE
getting error on hooking with the new PlaceholderAPI-2.10.7  

### ISSUE THREAD

**Issue by mihionmc** [DiscordSRV/DiscordSRV#819]
```
[00:00:43 INFO]: Enabling PlaceholderAPI hook
[00:00:43 WARN]: Task #2570 for DiscordSRV v1.19.2-SNAPSHOT generated an exception
java.lang.UnsupportedOperationException: PlaceholderAPI no longer provides a view of the placeholder's map!
Use: PlaceholderAPIPlugin.getInstance().getLocalExpansionManager().findExpansionByIdentifier(String)
        at me.clip.placeholderapi.PlaceholderAPI.getPlaceholders(PlaceholderAPI.java:159) ~[?:?]
        at me.clip.placeholderapi.PlaceholderAPI.getExpansions(PlaceholderAPI.java:207) ~[?:?]
        at github.scarsz.discordsrv.DiscordSRV.lambda$init$16(DiscordSRV.java:852) ~[?:?]
        at org.bukkit.craftbukkit.v1_16_R1.scheduler.CraftTask.run(CraftTask.java:99) ~[patched_1.16.1.jar:git-Paper-128]
        at org.bukkit.craftbukkit.v1_16_R1.scheduler.CraftScheduler.mainThreadHeartbeat(CraftScheduler.java:468) ~[patched_1.16.1.jar:git-Paper-128]
        at net.minecraft.server.v1_16_R1.MinecraftServer.b(MinecraftServer.java:1292) ~[patched_1.16.1.jar:git-Paper-128]
        at net.minecraft.server.v1_16_R1.DedicatedServer.b(DedicatedServer.java:377) ~[patched_1.16.1.jar:git-Paper-128]
        at net.minecraft.server.v1_16_R1.MinecraftServer.a(MinecraftServer.java:1213) ~[patched_1.16.1.jar:git-Paper-128]
        at net.minecraft.server.v1_16_R1.MinecraftServer.v(MinecraftServer.java:1001) ~[patched_1.16.1.jar:git-Paper-128]
        at net.minecraft.server.v1_16_R1.MinecraftServer.lambda$a$0(MinecraftServer.java:177) ~[patched_1.16.1.jar:git-Paper-128]
        at java.lang.Thread.run(Thread.java:834) [?:?]
```

**Comment by Vankka** [comment]
Please try with the latest development build - https://snapshot.discordsrv.com

**Comment by mihionmc** [comment]
Hi! I get the same resolut on the latest snapshot: 
I use java 11 if this makes any differens.
  
DiscordSRV-Build-1.19.2-SNAPSHOT-1596089057.jar

[08:06:33 INFO]: Enabling PlaceholderAPI hook
[08:06:33 WARN]: Task #2760 for DiscordSRV v1.19.2-SNAPSHOT generated an exception
java.lang.UnsupportedOperationException: PlaceholderAPI no longer provides a view of the placeholder's map!
Use: PlaceholderAPIPlugin.getInstance().getLocalExpansionManager().findExpansionByIdentifier(String)
        at me.clip.placeholderapi.PlaceholderAPI.getPlaceholders(PlaceholderAPI.java:159) ~[?:?]
        at me.clip.placeholderapi.PlaceholderAPI.getExpansions(PlaceholderAPI.java:207) ~[?:?]
        at github.scarsz.discordsrv.DiscordSRV.lambda$init$16(DiscordSRV.java:852) ~[?:?]
        at org.bukkit.craftbukkit.v1_16_R1.scheduler.CraftTask.run(CraftTask.java:99) ~[patched_1.16.1.jar:git-Paper-128]
        at org.bukkit.craftbukkit.v1_16_R1.scheduler.CraftScheduler.mainThreadHeartbeat(CraftScheduler.java:468) ~[patched_1.16.1.jar:git-Paper-128]
        at net.minecraft.server.v1_16_R1.MinecraftServer.b(MinecraftServer.java:1292) ~[patched_1.16.1.jar:git-Paper-128]
        at net.minecraft.server.v1_16_R1.DedicatedServer.b(DedicatedServer.java:377) ~[patched_1.16.1.jar:git-Paper-128]
        at net.minecraft.server.v1_16_R1.MinecraftServer.a(MinecraftServer.java:1213) ~[patched_1.16.1.jar:git-Paper-128]
        at net.minecraft.server.v1_16_R1.MinecraftServer.v(MinecraftServer.java:1001) ~[patched_1.16.1.jar:git-Paper-128]
        at net.minecraft.server.v1_16_R1.MinecraftServer.lambda$a$0(MinecraftServer.java:177) ~[patched_1.16.1.jar:git-Paper-128]
        at java.lang.Thread.run(Thread.java:834) [?:?]

**Comment by mihionmc** [comment]
Maybe it is placeholderAPI that has a problem because I get an error on supervanish plug too.

**Comment by mihionmc** [comment]
...okay they did it on purpose
https://www.spigotmc.org/resources/placeholderapi.6245/updates

**Comment by GreemDev** [comment]
Yeah, PAPI decided it'd be a good idea to push a *very* breaking change and only give developers 3 days to update their plugin(s) to support the change.

**Comment by Vankka** [comment]
Commited on the wrong branch, the latest snapshot contains the fix now https://snapshot.discordsrv.com (PAPI is also now planning on reverting some of the breaking changes in the next version)

**Comment by Vankka** [comment]
Fixed in release 1.20.0
